TelCo.ColorCoder/ManualMap.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+  class ManualMap{
+    
+    public static void ToString(){
```

Comment:
void Tostring() ?
should be string ToString()

Reply:
Yes, It should be string ToString().